src/features/subscriptions/components/PaymentPlanCard/PaymentPlanCard.tsx

```diff
@@ -78,7 +78,7 @@ const PaymentPlanCard = memo(function PaymentPlanCard({
         <div
           className={clsx(s.initialPrice, { [s.hidden]: billingOption.id === 'month' })}
         >
-          ${billingOption?.initialPricePerMonth}
+          ${billingOption?.initialPricePerMonth?.toLocaleString('en-US')} USD
```

Comment:
USD should probably be taken from translations strings (may be spelled differently in some languages)